include/mitsuba/render/shape.h

```diff
@@ -326,6 +326,8 @@ class MTS_EXPORT_RENDER Shape : public Object {
     inline Shape() { }
     virtual ~Shape();
 
+    /// Set this shape to its associated children
```

Comment:
Maybe "Explicitly register this shape as the parent of the provided sub-objects (emitters, sensors, etc.)"